doc/about.rst

```diff
@@ -38,23 +38,23 @@ citations to the following paper:
 If you want to cite scikit-learn for its API or design, you may also want to consider the
 following paper:
 
-`API design for machine learning software: experiences from the scikit-learn
-project <http://arxiv.org/abs/1309.0238>`_, Buitinck *et al.*, 2013.
-
-Bibtex entry::
-
-    @inproceedings{sklearn_api,
-      author    = {Lars Buitinck and Gilles Louppe and Mathieu Blondel and
-                   Fabian Pedregosa and Andreas Mueller and Olivier Grisel and
-                   Vlad Niculae and Peter Prettenhofer and Alexandre Gramfort
-                   and Jaques Grobler and Robert Layton and Jake VanderPlas and
-                   Arnaud Joly and Brian Holt and Ga{\"{e}}l Varoquaux},
-      title     = {{API} design for machine learning software: experiences from the scikit-learn
-                   project},
-      booktitle = {ECML PKDD Workshop: Languages for Data Mining and Machine Learning},
-      year      = {2013},
-      pages = {108--122},
-    }
+ `API design for machine learning software: experiences from the scikit-learn
```

Comment:
A single-character indent seems a strange choice... Maybe we should indent both this and the above a bit more so that it's more clearly indented.